test/e2e/README.md

```diff
@@ -9,6 +9,42 @@ To do this, following steps are recommended
 
 ocm get /api/clusters_mgmt/v1/clusters/(cluster-id)/credentials | jq -r .kubeconfig > /(path-to)/kubeconfig
 
-5. Run test suite using 
+5. AWS Credentials
+These are needed for interacting with the cluster. You can find them in the ~/.aws/credentials file.
+export AWS_ACCESS_KEY_ID=<your AWS access key ID>
+export AWS_SECRET_ACCESS_KEY=<your AWS secret access key>
+
+6. Generate OCM Token
+To authenticate with OCM, run the command below. It will return a token and also generate a configuration file.
+Run: ocm token
+Then export the token:
+export OCM_TOKEN=$(ocm token)
+
+7. Enable Debug Mode
+This enables detailed logging during test execution:
+export CAD_DEBUG=true
```

Comment:
What is this variable for if we're not running the cadctl afterwards?

Reply:
CAD_DEBUG? That variable doesn't even exist anymore, and it should have had no influence on e2e :)